src/js/claims-status/components/appeals-v2/Docket.jsx

```diff
@@ -2,60 +2,76 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import moment from 'moment';
 
-// TODO: Depending on when this gets rendered, we may want to return null if either
-//  `ahead` or `total` don't exist (or if `total` === 0 because of the divide by 0  error)
-function Docket({ ahead, total, form9Date }) {
-  const completedWidth = { width: `${((total - ahead) / total) * 100}%` };
-  const date = moment(form9Date, 'YYYY-MM-DD').format('MMMM YYYY');
+import DocketCard from './DocketCard';
+import { APPEAL_TYPES } from '../../utils/appeals-v2-helpers';
 
+/**
+ * @param {Number} ahead - The number of appeals ahead of this one
+ * @param {Number} total - The total number of appeals in the docket line
+ * @param {String} form9Date - The date the form 9 was sent in (or something)
+ * @param {String} docketMonth- The month that the board is looking at (or older)
+ * @param {String} appealType - The type of appeal
+ * @param {Bool}   aod - Whether the appeal is Advanced on Docket
+ * @param {Bool}   frontOfDocket - Whether the appeal is at the front of the docket
```

Comment:
Isn't it still called `front` until you rename it when you destructure the props? 🤷‍♂️

Reply:
Oh right. Good point. :+1:
I originally just had it destructured as `frontOfDocket` without renaming from `front`, but that...was broken for obvious reasons. After I fixed it, I forgot about this. Good catch!